pkg/insights/test/test_event_reader.go

```diff
@@ -10,10 +10,14 @@ func (t *TestEventReader) Recv() <-chan events.Event {
 	return t.Ch
 }
 
+func (t *TestEventReader) Close() {
+	close(t.Ch)
```

Comment:
I think `Subscribe().Close(); Subscribe().Close()` will panic because the channel will be double closed, right? If so maybe make a note here or something that this only works because there's only one call to `Subscribe` in the tests

Reply:
Added